### PR TITLE
Update entity-alias.mdx

### PR DESCRIPTION
**Pull request by jzck**
it was not clear for approle what the name should be the approle name or the role_id.

### CODE REVIEW

File: website/content/api-docs/secret/identity/entity-alias.mdx [hunk 1]
```diff
@@ -19,7 +19,8 @@ This endpoint creates a new alias for an entity.
 - `name` `(string: <required>)` - Name of the alias. Name should be the identifier
   of the client in the authentication source. For example, if the alias belongs
   to userpass backend, the name should be a valid username within userpass
-  backend. If alias belongs to GitHub, it should be the GitHub username.
+  backend. If alias belongs to GitHub, it should be the GitHub username. If alias 
+  belongs to an approle backend, the name should be a valid role_id.
```
Comment (yhyakuna) [inline website/content/api-docs/secret/identity/entity-alias.mdx:23]:
```suggestion
  auth method. If the alias belongs to GitHub, it should be the GitHub username. If the alias 
  belongs to an approle auth method, the name should be a valid RoleID.
```

**Review by yhyakuna — APPROVED** [review]
Thanks!

**Review by yhyakuna — APPROVED** [review]
Much better!

**Review by taoism4504 — APPROVED** [review]
LG

**Pull request merged**